api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/TxCurrDataset.java

```diff
@@ -81,7 +79,7 @@ public CohortIndicatorDataSetDefinition constructTxNewDatset(List<Parameter> par
 		 * Looks for patients enrolled in ART program (program 2=SERVICO TARV -
 		 * TRATAMENTO) before or on end date
 		 */
-		CohortDefinition inARTProgramDuringTimePeriod = sqlCohortQueries.getPatientsinARTProgramDuringTimePeriod();
+		CohortDefinition enrolledBeforeEndDate = inprogramCohortQueries.createInProgramParameterizableByDateAndLocation("InARTProgram", hivMetadata.getARTProgram(), Arrays.asList("onOrBefore"), "location");
```

Comment:
@rubailly is this cohort meant to only have one date parameter? `onOrBefore`

Aren't they all meant to have two `onOrBefore` and `onOrAfter`?

Reply:
Because it is tx_curr (..i.e they are looking for patients who are currently in the program), we only need the endDate parameter. You want just to check if the patient is still in the program at endDate